cli/preload_steps.go

```diff
@@ -0,0 +1,144 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/bitrise/configs"
+	"github.com/bitrise-io/bitrise/log"
+	stepman "github.com/bitrise-io/stepman/cli"
+	"github.com/bitrise-io/stepman/preload"
+	"github.com/urfave/cli"
+)
+
+const (
+	bitriseStepLibURL = "https://github.com/bitrise-io/bitrise-steplib.git"
+	bitriseMaintainer = "bitrise"
+)
+
+var stepsCommand = cli.Command{
+	Name:  "steps",
+	Usage: "Manage Steps cache.",
+	Subcommands: []cli.Command{
+		{
+			Name:  "list",
```

Comment:
```suggestion
			Name:  "list-cached",
```
It's a bit confusing without knowing about offline mode

Reply:
Renamed.